### DISCUSSION TITLE
"Collision on the bus, retrying..." after power outage

### DISCUSSION THREAD

**Discussion by DavidS** [fredlcore/BSB-LAN#662] [Q&A]
Hi,

since a recent power outage, by BSB-LAN (via Arduino Due) device won't talk to the BSB bus anymore. The best I could get out of it is "Collision on the bus, retrying..." after upgrading the v3.2.2 I've been running successfully before to the latest version off git on the serial console (and also https://github.com/fredlcore/BSB-LAN/issues/661). The status LED is lit dimly, no flickering. Disconnecting the bus from the controller extinguishes the LED. The room unit still works. I've reset both the heating controller and the BSB-LAN device several times in all ways I could think of. The device has worked fine for the last few years.

Any idea what else I could try?

Cheers, David

**Comment by fredlcore** [comment]
Depending on the reasons of the power outage either the heating system or the BSB-LAN setup could have been damaged. A non-flickering LED means that there is no communication on the bus. To identify the reasons for this from remote is basically impossible.
